dashboard/config/routes.rb

```diff
@@ -149,6 +149,16 @@ module OPS
       end
     end
 
+    # /s/xxx/lockable/yyy/puzzle/zzz
+    resources :lockable_stages, only: [], path: "/lockable", format: false do
```

Comment:
If you want to change `params[:lockable_stage_id]` in `ScriptLevelsController#load_script_level` to something more appropriate you could set `param: num` here (see: http://stackoverflow.com/a/13427336).